commands/cli.go

```diff
@@ -111,24 +115,55 @@ func awsNuke(c *cli.Context) error {
 	if err != nil {
 		return errors.WithStackTrace(err)
 	}
+
+	selectedRegions := c.StringSlice("region")
 	excludedRegions := c.StringSlice("exclude-region")
 
+	if len(selectedRegions) > 0 && len(excludedRegions) > 0 {
```

Comment:
Logic seems good to me, but I wonder, for code cleanliness purposes, if this region filtering process shouldn't happen in an auxiliary function?

Reply:
Thanks for the feedback @Etiene  - makes sense. I moved the region filtering process to a separate function ```aws.GetTargetRegions``` which in turn also helped to create tests for the same in ```aws_test.go``` as ```TestGetTargetRegions```

Reply:
Hi @Etiene - just wanted to check if this is fine and can be merged? Thanks.